src/components/NewMessage/NewMessage.vue

```diff
@@ -354,6 +357,8 @@ export default {
 				return t('spreed', 'No permission to post messages in this conversation')
 			} else if (!this.currentConversationIsJoined) {
 				return t('spreed', 'Joining conversation …')
+			} else if (this.silentChat) {
+				return t('spreed', 'Write a message (silent)')
```

Comment:
```suggestion
				return t('spreed', 'Write a message without notification')
```

as we don't use "silent" ?